README.md

```diff
@@ -58,7 +58,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Design](#design)
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
-  - [Programming Courses Free Online](#programming-courses-free-online)
+  - [SoloLearn](#programming-courses-free-online)
```

Comment:
This one should be the title of the section (Programming Courses Free Online). What's under that section should be the title of the resource (in this case, SoloLearn).

Reply:
I have Changed Title of content to "Programming Courses Free Online" and section title the same.
I think it is fine now